JASP-Engine/JASP/R/packagecheck.R

```diff
@@ -350,11 +350,14 @@
 }
 
 
-.initPackages <- function(installed) {
+.initPackages <- function(installed.packages) {
```

Comment:
The variable name might be a bit inconvenient: it's kind of clashy with utils::installed.packages. Perhaps a better name would be `installedPackages`?